src/database.tf

```diff
@@ -128,7 +128,7 @@ resource "azurerm_cosmosdb_mongo_database" "transaction" {
 
   name                = "transaction"
   resource_group_name = azurerm_resource_group.db_rg.name
-  account_name        = module.cosmosdb_account_mongodb[0].name
+  account_name        = module.cosmosdb_account_mongodb[count.index].name
```

Comment:
it's ok, but since count can only assumes 0 or 1 .... it does not change that much.
It's a style exercise ;-)

Reply:
@GiovanniMancini  mmmm well I changed my mind...
When count = 0 the resource **azurerm_cosmosdb_mongo_database.transaction** does not exist
while when count = 1 the resource exists but I feel module.cosmosdb_account_mongodb[1].name is throwing an error: the index 1 does not exist.
Am I wrong?